action/Scanning.action

```diff
@@ -2,24 +2,21 @@
 
 # Pre-defined constants to code the type of scanning action.
 int8 UNKNOWN_TYPE=0  # default value does not code a valid command
-int8 SHELF=1  # detect and localize shelf floors
-int8 BAR_CODE=2  # detect, recognize, and localize bar codes
-int8 SEPARATOR=3  # detect and localize separators
-int8 OBJECT_RECOGNITION=4  # detect, recognize, and localize front object in facing
-int8 OBJECT_COUNTING=5  # count all objects within one facing
+int8 COMPLETE_SCAN=1  # perform all valid scanning steps
 
 # Requested type of scanning action, fill with one of the above constants.
 int8 type
 
-# List of locations that shall be scanned.
-refills_msgs/Location[] locations
+# List of locations that shall be scanned, encoded by their unique IDs,
+# as defined in the semantic store maps.
+string[] loc_id
```

Comment:
with location you mean shelf or can it also refer to racks?

Reply:
Correct. I extended the comment to be more specific.